### PR TITLE
chore: Add eslint and mergeable

### PR DESCRIPTION
**Pull request by jacksonjp**
# Add mergeable config

## What are you adding?
Forces PR titles to be consistent, following are valid titles

```
chore: Update something
fix(ISSUE-123): Fix a bug
feature: Added new feature
```

### CODE REVIEW

File: .github/mergeable.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: 2
+mergeable:
+  - when: pull_request.*
+    validate:
+      - do: title
+        must_exclude:
+          regex: ^WIP
+        must_include:
+          regex: ^((feature|chore|fix)(\(.{1,15}\))?|()\-[0-9]{1,5})\:.+$
```
Comment (yukinoda) [inline .github/mergeable.yml:9]: How about following the types in the [contribution guide](https://github.com/monstar-lab-oss/admin-panel-template-reactjs/blob/main/CONTRIBUTING.md)?

```suggestion
          regex: ^((feat|fix|docs|style|refactor|perf|test|build|ci|chore)(\(.{1,15}\))?|()\-[0-9]{1,5})\:.+$
```

**Review by yukinoda — COMMENTED** [review]
left one suggestion
let's see if we can figure out fixing netlify's issue

**Review by yukinoda — APPROVED** [review]
mergeable working fine 🎉  Thanks!

**Review by geekonthemove — APPROVED** [review]
Looks good 👍

**Pull request merged**